Automattic/jetpack
# Repo Gardening: add task to automate "Triaged" board action

> **Warning**
> This should not be merged until #33469 gets merged.

Fixes 3660-gh-dotcom-forge

## Proposed changes:

In this PR, we extend on the structure added in #33469. We now also watch for the "Triaged" label on an issue. If that label is added, the "Status" field of the GitHub Project board will be updated accordingly.

### Other information:


## Jetpack product discussion

* 3660-gh-dotcom-forge

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

This cannot be tested in this repo, but you can check the results in a fork of this repo (see the comments with screenshots for details):
https://github.com/jeherve/jetpack/issues/101
